data/frame_meta.go

```diff
@@ -44,6 +50,15 @@ func (f *Frame) AppendNotices(notices ...Notice) {
 	f.Meta.Notices = append(f.Meta.Notices, notices...)
 }
 
+// QueryResultMetaStat matches https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/data.ts#L53
+type QueryResultMetaStat struct {
+	FieldConfig
+
+	DisplayName string `json:"displayName"`
+
+	Value float32 `json:"value"`
```

Comment:
Should maybe be a pointer? This part of the object is serialized into JS, so "NaN/Inf" values will not be supported. But if it is a pointer, it is nullable, and therefore could have a value of null.

Reply:
no need for null... if it is null the whole element should not be added